sdk/go/x/auto/example_test.go

```diff
@@ -1073,3 +1075,98 @@ func ExampleStack_SetAllConfig() {
 	// set all config in map
 	_ = stack.SetAllConfig(ctx, cfg)
 }
+
+func ExampleStack_Cancel() {
+	ctx := context.Background()
+	stackName := FullyQualifiedStackName("org", "project", "stack")
```

Comment:
we should show an example for local backends that don't use the FQSN

Reply:
Good point. I'd like to invert the solution a little though. Rather than having full coverage of examples for the caveats of SaaS vs non-saas backends. Let's keep all examples as is assuming usage of SaaS backend, and invest in a few end to end high polish examples for local backends (Stack and LocalWorkspace level examples that show the end to end workflow). 

I've captured this here: https://github.com/pulumi/pulumi/issues/5381
Perhaps @stack72 might be able to take a look as a part of the OSS backend audit.